### PR TITLE
Feature/3495

### PR DESCRIPTION
**Pull request by brammiedema**
Added ExAC annotator in new framework format.

### CODE REVIEW

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/resources/impl/MultiFileResource.java [hunk 1]
```diff
@@ -98,4 +103,20 @@ public String getName()
 		return resource.findAll(q);
 	}
 
+<<<<<<< HEAD
```
Comment (fdlk) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/resources/impl/MultiFileResource.java]: conflict needs merging

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/impl/ExacAnnotator.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+		repoMetaData.addAttributeMetaData(new DefaultAttributeMetaData("EA_MAF", DECIMAL));
+
+		repoMetaData.addAttribute("id").setIdAttribute(true).setVisible(false);
+
+		exacTabixResource = new ResourceImpl(EXAC_TABIX_RESOURCE, new SingleResourceConfig(EXAC_FILE_LOCATION_PROPERTY,
+				molgenisSettings), new TabixRepositoryFactory(repoMetaData));
+
+		return exacTabixResource;
+	}
+}
```
Comment (fdlk) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/impl/ExacAnnotator.java]: WHOO!!

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/CmdLineAnnotator.java [hunk 3]
```diff
@@ -91,6 +91,18 @@ public void run(String[] args) throws Exception
...
+
+		PrintWriter outputVCFWriter = new PrintWriter(outputVCFFile, "UTF-8");
+
+		VcfRepository vcfRepo = new VcfRepository(inputVcfFile, this.getClass().getName());
+		Iterator<Entity> vcfIter = vcfRepo.iterator();
+
+		// VcfUtils.checkPreviouslyAnnotatedAndAddMetadata(inputVcfFile, outputVCFWriter, infoFields, CADD_SCALED);
+
+		System.out.println("Now starting to process the data.");
+
```
Comment (bartcharbon) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/CmdLineAnnotator.java]: What is the purpose of reintroducing this code?

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/CmdLineAnnotator.java [hunk 4]
```diff
@@ -102,6 +114,14 @@ else if (annotatorName.equals("snpeff"))
...
 			new SnpEffServiceAnnotator(annotationSourceFile, inputVcfFile, outputVCFFile);
 		}
+		else if (annotatorName.equals("dann"))
+		{
+			//new DannAnnotator(annotationSourceFile, inputVcfFile, outputVCFFile);
+		}
+		else if (annotatorName.equals("fitcon"))
+		{
+			//new FitconAnnotator(annotationSourceFile, inputVcfFile, outputVCFFile);
+		}
```
Comment (bartcharbon) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/CmdLineAnnotator.java]: Remove lines for non-existent annotators

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/impl/DannAnnotator.java [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+//	}
+//
+//	@Override
+//	public List<AttributeMetaData> getOutputMetaData()
+//	{
+//		// TODO Auto-generated method stub
+//		return null;
+//	}
+//
+//}
```
Comment (bartcharbon) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/impl/DannAnnotator.java]: Remove this class from the pull request

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/impl/FitconAnnotator.java [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+//			{
+//				if (tabixReader == null)
+//				{
+//					tabixReader = new TabixReader(molgenisSettings.getProperty(CADD_ANNOTATED_FILE_LOCATION_PROPERTY));
+//				}
+//			}
+//		}
+//	}
+//
+//}
```
Comment (bartcharbon) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/impl/FitconAnnotator.java]: Remove this class from the pull request

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/CmdLineAnnotator.java (hunk 3 above)
Comment (brammiedema) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/CmdLineAnnotator.java]: artifact from merging with my existing branch

**Pull request closed without merging**